src/create-link/Dockerfile

```diff
@@ -6,4 +6,7 @@ COPY . /code
 
 WORKDIR /code
 
+# save created snippet variables to `/workdir/subdomain-domain-tld.env`
+VOLUME /workdir
```

Comment:
do we need this? 

this will store link env vars in a anonymous volume if /workdir isn't explicitly mounted, right?

Reply:
Yes, without it, the directory at `/workdir` won't exist and the entrypoint will fail to create the `/workdir/*.env` file. It could be replaced with a `mkdir /workdir` somewhere, but then, there won't be any way to retrieve the `.env` file when the container exit (if not mounted).